integration-tests/features/steps/s3_component_analysis.py

```diff
@@ -68,12 +69,39 @@ def check_component_core_data(context, package, version, ecosystem):
                            "package_info", "subtasks"]
     check_attributes_presence(data, attributes_to_check)
 
-    # check the attributes for the 'analyses' subnode
-    analyses = data["analyses"]
-    attributes_to_check = ["security_issues", "metadata", "keywords_tagging",
-                           "digests", "source_licenses", "dependency_snapshot"]
+    # NOTE: 'analyses' subnode has to be checked in explicit test steps
 
-    check_attributes_presence(analyses, attributes_to_check)
+
+def _node_items_to_check(context, items, node):
+    expected_items = split_comma_separated_list(items)
+    assert expected_items is not None
+
+    data = context.s3_data
+    returned_items = check_and_get_attribute(data, node)
+    assert returned_items is not None
+
+    return expected_items, returned_items
```

Comment:
I'd perhaps change the order to `return returned_items, expected_items` as you're later passing those to `check_attributes_presence(returned_items, expected_items)`

Reply:
I'll fix it, TYVM